README.md

```diff
@@ -0,0 +1,18 @@
+# Lazy
+
+A library for lazy objects, started life as a coredata/contrib package.
+
+## Release
+
+Assuming you use virtualenvwrapper, you need to set up your environment,
+prior to being able to release:
+
+    mkvirtualenv lazy
+    pip install -r release-requirements.txt
```

Comment:
Q: aren't we generally moving requirements into a requirements folder, so this should go under `requirements/release.txt` ?